newIDE/app/src/ObjectGroupsList/index.js

```diff
@@ -276,6 +281,30 @@ export default class GroupsListContainer extends React.Component<Props, State> {
     this.sortableList.getWrappedInstance().forceUpdateGrid();
   };
 
+  _setAsGlobalGroup = (groupWithContext: GroupWithContext) => {
+    const { group } = groupWithContext;
+    const { globalObjectGroups, objectGroups } = this.props;
+
+    const groupName = group.getName();
+
+    if (globalObjectGroups.has(groupName)) {
+      showWarningBox(
+        'A global object with this name already exists. Please change the object name before setting it as a global object'
+      );
+      return;
+    }
+
+    //eslint-disable-next-line
+    const answer = confirm(
+      "This group will be loaded and available in all the scenes. This is only recommended for groups that you reuse a lot and can't be undone. Make this group global?"
+    );
+    if (!answer) return;
+
+    globalObjectGroups.insertNew(groupName, globalObjectGroups.count());
```

Comment:
This does not seem to work: you're inserting a new empty group (insertNew). Instead, you should use this method Insert:
https://github.com/4ian/GDevelop/blob/f478c861aff417a8aca8909db4c943d6bcdbc229/GDevelop.js/Bindings/Bindings.idl#L212

passing it the group (instead of the groupName).
Try and let me know if it works better.

Reply:
How you see it's not working ?
I've changed the method :)

Reply:
Just by reading the code ;) I could also patch the changes on my branch and give it a try, but as the changes are fairly small reading should be sufficient to detect these things (and as possible I try to read as much possible of the code to ensure all best practices and choices are made).